subsys/bluetooth/controller/ll_sw/ull_central_iso.c

```diff
@@ -228,19 +229,19 @@ uint8_t ll_cig_parameters_commit(uint8_t cig_id, uint16_t *handles)
 		 * handle the throughput. For unframed these must be divisible, if they're not,
 		 * framed mode must be forced.
 		 */
-		cig->iso_interval = cig->c_sdu_interval / ISO_INT_UNIT_US;
+		iso_interval_us = cig->c_sdu_interval;
 
-		if (cig->iso_interval < BT_HCI_ISO_INTERVAL_MIN) {
+		if (iso_interval_us < 5000U) {
 			/* ISO_Interval is below minimum (5 ms) */
-			cig->iso_interval = BT_HCI_ISO_INTERVAL_MIN;
+			iso_interval_us = 5000U;
```

Comment:
Suggest to keep the use of the defined HCI values but add a macro or something similar to 
```c
/**
 * @brief Convert periodic advertising interval (N * 1.25 ms) to milliseconds
 *
 * 5 / 4 represents 1.25 ms unit.
 */
#define BT_GAP_PER_ADV_INTERVAL_TO_MS(interval) ((interval) * 5 / 4)
```

like
```c
#define HCI_ISO_INTERVAL_TO_US(_interval) (USEC_PER_MSEC * (_interval) * 5 / 4)
```

Edit: Just noticed `ISO_INT_UNIT_US`, so this could also be 
```c
#define HCI_ISO_INTERVAL_TO_US(_interval) ((_interval) * ISO_INT_UNIT_US)
```

Reply:
Updated.